go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/onsi/gomega v1.24.2
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
+	github.com/vertica/vcluster v0.0.0-20230626111548-d9d8c0dd50db
```

Comment:
@spilchen is there a way to have a more readable version here?

Reply:
Once changes in vcluster slow down, we can create an offiical GitHub release for that repo. We can then use the version of that release here instead of a gitRef.